### ISSUE TITLE
Add tests for Keras callbacks: MetricAverageCallback, LearningRateScheduleCallback and LearningRateWarmupCallback

### ISSUE THREAD

**Issue by ChaiBapchya** [horovod/horovod#2659]
**Is your feature request related to a problem? Please describe.**
Currently in tensorflow2_keras .py only broadcastglobalvariablescallback is tested.

**Describe the solution you'd like**
Add similar tests for other 3 callback APIs

**Describe alternatives you've considered**
NA

**Additional context**
NA

**Comment by tgaddair** [comment]
Good idea @ChaiBapchya.  Feel free to create a PR if you have time to work on this.
